arrow-docs/docs/arrow/typeclasses/profunctor/README.md

```diff
@@ -61,18 +47,6 @@ Contramap on the first type parameter and map on the second type parameter.
 
 `fun Kind2<F, A, B>.dimap(fl: (C) -> A, fr: (B) -> D): Kind2<F, C, D>`
 
-```kotlin:ank
```

Comment:
Here and above we would need some other examples

Reply:
This typeclass will be removed in `0.13.0` and is deprecated in next version. I don't think we should invest more time on something that is planned to be removed soon.